casparser/process/regex.py

```diff
@@ -34,4 +34,5 @@
 # Tax transactions
 TRANSACTION_RE3 = rf"{date_re}\t\t([^0-9].*)\t\t{amt_re}(?:\t\t{amt_re}\t\t{amt_re}\t\t{amt_re})*"
 DESCRIPTION_TAIL_RE = r"(\n.+?)(\t\t|$)"
+TRANSACTION_RE4 = rf"{date_re} (\*+[^0-9].*\*+)(?:\t\t{amt_re}\t\t{amt_re}\t\t{amt_re}\t\t{amt_re})*"
```

Comment:
The above regex is not working for any CAS pdf files I have. It may require modification as follows. Please check. 

```suggestion
TRANSACTION_RE4 = rf"{date_re}\t\t(\*+[^0-9].*\*+)(?:\t\t{amt_re}\t\t{amt_re}\t\t{amt_re}\t\t{amt_re})*"
```